src/libecalc/core/result/base.py

```diff
@@ -33,7 +33,16 @@ def extend(self, other: Self) -> Self:
                 # In case of nested models such as compressor with turbine
                 values.extend(other_values)
             elif isinstance(values, list):
-                if isinstance(other_values, list):
+                # Temporary v2 only
+                # We are not able to currently import PumpModelResult due to circular import, therefore we use duck typing...
+                if (
+                    len(values) == 1
+                    and len(other_values) == 1
+                    and hasattr(values[0], "extend")
+                    and hasattr(other_values[0], "extend")
+                ):
+                    values[0].extend(other_values[0])
```

Comment:
Why was this added?

Reply:
Due to the comment above...circular import. Next step with v2 will have to be to sort out a few ofthe ciruclar import issues..